src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,29 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.info("Method completeOrder was called for userId: " + userId);
         List<Product> products = getAllProductsFromShoppingCart(userId);
```

Comment:
Ensure that the `getAllProductsFromShoppingCart` method is correctly implemented to fetch products from a real data source. Currently, it returns a hardcoded list of products, which is fine for testing but should be replaced with actual data retrieval logic in a production environment.